core-services/prow/02_config/openshift/assisted-image-service/_pluginconfig.yaml

```diff
@@ -11,7 +11,7 @@ bugzilla:
           branches:
             main:
               target_release: rhacm-2.6
-            release-ocm-2.4:
```

Comment:
Doesn't it break out option to backport fixes to 2.4 branch?

Reply:
I think you right. I'll add it back